navit/plugin.c

```diff
@@ -1,4 +1,4 @@
-/**
+/*
  * Navit, a modular navigation system.
  * Copyright (C) 2005-2008 Navit Team
```

Comment:
No big deal for this one, but we could probably update the (c) statement to a current year when we edit a file.